backend/dev.Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:12.7.0
```

Comment:
@kawcl Can you try once with nodejs 16 image using `FROM node:16.3.0-alpine`. I have checked our Upgrade for nodejs 16 locally and I works fine, just we need to upgrade the "pg" package in backend/packages/Upgrade/package.json from "^7.12.1" to "^8.3.0". Can you do these two changes for backend and the `FROM node:16.3.0-alpine` change for frontend and test if the local development works well?

Reply:
I can make these changes and try getting things to run.  I'll be using a non-alpine node image just because of how I set up docker locally for development. I don't want to use the minimized OS.

Reply:
I would strongly recommend using the same OS in your dev environment and in deployments. There are so many times where I run into trouble because something in alpine works very differently than ubuntu and I waste a lot of time trying to track down the problem.

Is there something you need for development that alpine doesn't have?

Reply:
no not necessarily. it's just easier to not work with alpine since I'm not as familiar with it but I can try to get it running with alpine, it hopefully shouldn't be too much work to make the switch.

Reply:
update to 12.7.0-alpine was more flawless than I expected. I'll try the update with 16.3.0-alpine